### ISSUE TITLE
Update BullBitcoin's Wasabi-Coinjoin support guides

### ISSUE THREAD

**Issue by yahiheb** [zkSNACKs/WasabiDoc#1286]
Check all the following guides and update their content and their visuals (logo):

- https://www.bitcoinsupport.com/ --> Section: Privacy and Coinjoin
- https://guides.bitcoinsupport.com/guides/v/wasabi/ --> https://github.com/SatoshiPortal/wasabiwallet-guide (this one is more up to date)
- https://guides.bitcoinsupport.com/guides/v/coinjoin/ --> https://github.com/SatoshiPortal/coinjoin-guide

Either open pull requests to update the guides, or at least open issues in their repositories.
Whoever takes this issue, please tag me in the PRs to help review them.

Some already open issues:

https://github.com/SatoshiPortal/wasabiwallet-guide/issues/7
https://github.com/SatoshiPortal/wasabiwallet-guide/issues/8
https://github.com/SatoshiPortal/wasabiwallet-guide/issues/9

https://github.com/SatoshiPortal/coinjoin-guide/issues/1

cc @MarnixCroes @BTCparadigm @jesseaam

**Comment by yahiheb** [comment]
Once this is done, maybe @thibistaken can help update the french version of those guides:

- https://www.bitcoinsupport.com/fr --> Confidentialité et CoinJoin
- https://guides.bitcoinsupport.com/guides-fr/v/portefeuille-wasabi/ --> https://github.com/SatoshiPortal/wasabiwallet-guide-fr
- https://guides.bitcoinsupport.com/guides-fr/v/coinjoin-avec-wasabi/ --> https://github.com/SatoshiPortal/coinjoin-guide-fr

**Comment by jesseaam** [comment]
### Specifics

> https://www.bitcoinsupport.com/ --> Section: Privacy and Coinjoin

I emailed their support to ask where the GitHub repo was for this page. I described that I wanted to update this section. The two main things I see is that the sentence `...we will teach you the art of “coin selection” ...` should be updated. And the WW logo should be updated. The logo should also be updated in the section: Desktop Wallet (Wasabi). 

<br><br>

> https://guides.bitcoinsupport.com/guides/v/wasabi/ --> https://github.com/SatoshiPortal/wasabiwallet-guide (this one is more up to date)

The main thing I see that needs updated here is the logo. If we want to be fussy we could request them to change the sentence `It is easy to use once you overcome a small learning curve.` to `It is intuitive and easy to use.` Since IMO WW2.0 has no more of a learning curve than any other Bitcoin wallet. Furthermore, we could request to add the sentence `Wasabi Wallet has excellent documentation, tutorials, and a responsive support team.`since it's true :)


<br><br>

> https://guides.bitcoinsupport.com/guides/v/coinjoin/ --> https://github.com/SatoshiPortal/coinjoin-guide

Again, the logo needs updated. The sentence `Using CoinJoin with Wasabi requires a minimum of 0.11 bitcoin.` needs updated to reflect the new minimum of 5000 sats ([ref](https://docs.wasabiwallet.io/FAQ/FAQ-UseWasabi.html#what-is-the-minimum-amount-required-to-coinjoin)).

**Comment by yahiheb** [comment]
> Also, I found another page on their website that needs completely rewritten: https://guides.bitcoinsupport.com/guides/v/coinjoin/using-conjoin/using-coinjoin

I have mentioned https://guides.bitcoinsupport.com/guides/v/coinjoin/ in the description and all the pages of that guide should be updated if needed.

There is this example for labelling in https://guides.bitcoinsupport.com/guides/v/coinjoin/using-coinjoin/overview:
`So for example, if you receive bitcoin as payment for a job you completed, you can write “Bull Bitcoin pay”.`

IMO the label should simply be `“Bull Bitcoin”` not `“Bull Bitcoin pay”`.

**Comment by jesseaam** [comment]
I created another PR: https://github.com/SatoshiPortal/coinjoin-guide/pull/5

**Comment by jesseaam** [comment]
PR: [update wasabi wallet logo in this image #7](https://github.com/SatoshiPortal/coinjoin-guide/pull/7
)

**Comment by yahiheb** [comment]
> PR: [update wasabi wallet logo in this image #7](https://github.com/SatoshiPortal/coinjoin-guide/pull/7)

This one is merged.

**Comment by yahiheb** [comment]
This is obsolete, I cannot find those guides anymore.
